raiden/transfer/events.py

```diff
@@ -374,6 +374,78 @@ def from_dict(cls, data: Dict[str, Any]) -> "ContractSendChannelBatchUnlock":
         return restored
 
 
+class ContractSendChannelBatchUnlockLight(ContractSendEvent):
+    """ Event emitted when the lock must be claimed on-chain. """
+
+    def __init__(
+        self,
+        canonical_identifier: CanonicalIdentifier,
+        client: Address,
+        participant: Address,
+        triggered_by_block_hash: BlockHash,
+    ) -> None:
+        super().__init__(triggered_by_block_hash)
+        self.canonical_identifier = canonical_identifier
+        self.participant = participant
+        self.client = client
+
+    @property
+    def token_network_identifier(self) -> TokenNetworkAddress:
+        return TokenNetworkAddress(self.canonical_identifier.token_network_address)
+
+    @property
+    def channel_identifier(self) -> ChannelID:
+        return self.canonical_identifier.channel_identifier
+
+    def __repr__(self) -> str:
+        return (
+            "<ContractSendChannelBatchUnlock token_network_id:{} "
+            "channel:{} participant:{} triggered_by_block_hash:{} client{}"
+            ">"
+        ).format(
+            pex(self.token_network_identifier),
+            self.channel_identifier,
+            pex(self.participant),
+            pex(self.triggered_by_block_hash),
+            pex(self.client),
+        )
+
+    def __eq__(self, other: Any) -> bool:
+        return (
+            super().__eq__(other)
+            and isinstance(other, ContractSendChannelBatchUnlockLight)
+            and self.canonical_identifier == other.canonical_identifier
+            and self.participant == other.participant
+            and self.client == other.client
+        )
+
+    def __ne__(self, other: Any) -> bool:
+        return not self.__eq__(other)
+
+    def to_dict(self) -> Dict[str, Any]:
+        result = {
+            "canonical_identifier": self.canonical_identifier.to_dict(),
+            "participant": to_checksum_address(self.participant),
```

Comment:
Be careful with the address here you should use to_normalized_address as i put on the comment above